BitmovinAnalyticsCollector/Classes/data/EventData.swift

```diff
@@ -55,6 +55,8 @@ public class EventData: Codable {
     var pageLoadType: Int = 1
     var pageLoadTime: Int64 = 0
     var version: String?
+    var sequenceNumber: Int32 = 0
+    var platform: String = "iOS"
```

Comment:
I assume we want to support tvOS here as well? If so, you can use this macro to detect 

```swift
#if os(iOS)
// compiles for iOS
#elseif os(tvOS)
// compiles for tvOS
#endif
```

Reply:
Thanks implemented :)